.gitignore

```diff
@@ -1,29 +1,30 @@
-**/venv/
-**/.idea/
-
-src/gui/node_modules/
-src/gui/dist/
-src/core/__pycache__/
-src/core/migrations/versions/__pycache__/
+# built and cached files
+venv/
+__pycache__/
+*.pyc
+node_modules/
+dist/
+build/
 
+# temporary files of editors
 .DS_Store
-.*.swp
```

Comment:
The proper pattern would be `.*.sw?`, not `*.sw?`

Reply:
fixed